src/AutoMapper.Extensions.Microsoft.DependencyInjection/AutoMapper.Extensions.Microsoft.DependencyInjection.csproj

```diff
@@ -21,8 +21,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="[3.0.0, 4.0)" />
-    <PackageReference Include="Microsoft.Extensions.Options" Version="[3.0.0, 4.0)" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="[3.0.0, 5.0)" />
```

Comment:
@jbogard I think we should rollback to >= here. MS is not going to break compatibility.

Reply:
So just make it 3.0?

Reply:
Yes, rather than keep changing it.

Reply:
Works for me. Let's change it.

Reply:
Done.

Reply:
All looks good I believe :)